Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/main/home/HomeFragmentViewModel.kt

```diff
@@ -321,6 +343,9 @@ class HomeFragmentViewModel @AssistedInject constructor(
         cwaSettings.wasTracingExplanationDialogShown = true
     }
 
+    private fun isOlderThanThreshold(hours: Duration, testTimestamp: Instant): Boolean =
+        Days.daysBetween(testTimestamp, Instant.now()).toStandardDuration().standardHours >= hours.standardHours
```

Comment:
btw , I think you can compare durations directly no need to get hours 
`toStandardDuration()>= hours`